ReactiveCocoa.podspec

```diff
@@ -17,10 +17,10 @@ Pod::Spec.new do |s|
   s.source       = { :git => "https://github.com/ReactiveCocoa/ReactiveCocoa.git", :tag => "#{s.version}" }
   s.source_files = "ReactiveCocoa/*.{swift,h,m}"
   s.private_header_files = "ReactiveCocoa/RACObjCRuntimeUtilities.h"
-  s.osx.source_files = "ReactiveCocoa/AppKit/*.{swift}"
-  s.ios.source_files = "ReactiveCocoa/UIKit/*.{swift}"
+  s.osx.source_files = "ReactiveCocoa/AppKit/*.{swift}", "ReactiveCocoa/Shared/*.{swift}"
+  s.ios.source_files = "ReactiveCocoa/UIKit/*.{swift}", "ReactiveCocoa/Shared/*.{swift}"
   s.tvos.source_files = "ReactiveCocoa/UIKit/*.{swift}"
-  s.tvos.exclude_files = "ReactiveCocoa/UIKit/*{UIDatePicker,UISwitch}*"
+  s.tvos.exclude_files = "ReactiveCocoa/UIKit/*{UIDatePicker,UISwitch}*", "ReactiveCocoa/Shared/*.{swift}"
```

Comment:
Oh, I'm sorry I made a mistake here. `NSLayoutConstraint.swift` should not be excluded for tvOS since the file is added to `ReactiveCocoa-tvOS` target in the Xcode project. I meant excluding from watchOS source files. 🙇